staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2650,6 +2660,8 @@ type VirtualMachineInstanceGuestAgentInfo struct {
 	// It will be set to "frozen" if the request was made, or unset otherwise.
 	// This does not reflect the actual state of the guest filesystem.
 	FSFreezeStatus string `json:"fsFreezeStatus,omitempty"`
+	// Load contains the system load averages (1M, 5M, 15M) from the guest agent
+	Load VirtualMachineInstanceGuestOSLoad `json:"load,omitempty"`
```

Comment:
@machadovilaca This should be a pointer IMO, so that we will be backward-compatible. In case a new virt-handler speaks to old virt-launcher (can happen during upgrades) we will expect nil here.